src/video_core/amdgpu/liverpool.cpp

```diff
@@ -544,90 +558,99 @@ Liverpool::Task Liverpool::ProcessCompute(std::span<const u32> acb, int vqid) {
     while (!acb.empty()) {
         const auto* header = reinterpret_cast<const PM4Header*>(acb.data());
         const u32 type = header->type;
-        if (type != 3) {
-            // No other types of packets were spotted so far
-            UNREACHABLE_MSG("Invalid PM4 type {}", type);
-        }
 
-        const u32 count = header->type3.NumWords();
-        const PM4ItOpcode opcode = header->type3.opcode;
-        const auto* it_body = reinterpret_cast<const u32*>(header) + 1;
-        switch (opcode) {
-        case PM4ItOpcode::Nop: {
-            const auto* nop = reinterpret_cast<const PM4CmdNop*>(header);
+        switch (type) {
```

Comment:
Please return it back. Packet type 2 is not used on ASC queue (even for alignment cases as GFX has)